tests/test_server.py

```diff
@@ -210,6 +212,32 @@ def test_predict(http_app, app):
     assert response.status_code == 200
 
 
+def test_evaluate(app):
+    with io.open(DEFAULT_STORIES_FILE, 'r') as f:
+        stories = f.read()
+    response = app.post('/evaluate',
+                        data=stories)
+    assert response.status_code == 200
+    assert set(response.get_json().keys()) == {"report",
+                                               "precision",
+                                               "f1",
+                                               "accuracy",
+                                               "in_training_data_fraction"}
+
+
+def test_end_to_end_evaluation(app):
+    with io.open(END_TO_END_STORY_FILE, 'r') as f:
+        stories = f.read()
+    response = app.post('/evaluate?e2e=true',
+                        data=stories)
+    assert response.status_code == 200
+    assert set(response.get_json().keys()) == {"report",
```

Comment:
this isn't really testing if e2e is working - the response test is the same as the above

Reply:
Only way to know really is to add field to the return dict of `rasa_core.evaluate`